recipes/r-ggjoy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '0.4.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ggjoy
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: ggjoy_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/ggjoy_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/ggjoy/ggjoy_{{ version }}.tar.gz
+  sha256: cb9ef790921ffcd3cfb6a55b409d17ccae9e8f5fdd2a28e55ea2ccfa8efd44e8
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-ggplot2 >=2.2.0
+    - r-ggridges >=0.4.0
+  run:
+    - r-base
+    - r-ggplot2 >=2.2.0
+    - r-ggridges >=0.4.0
+
+test:
+  commands:
+    - $R -e "library('ggjoy')"  # [not win]
+    - "\"%R%\" -e \"library('ggjoy')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/web/packages/ggjoy/index.html
```

Comment:
Not a huge deal, but when you have to manually fix the home URL, it's best to use the URL listed on the CRAN page (which is what the cran skeleton normally does when working correctly). In this case that would have been https://github.com/clauswilke/ggjoy